src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,54 @@
+package duke.exception;
+
+/**
+ * Exception representing invalid commands fed to application.
+ *
+ * @author WR3nd3
+ */
+public class DukeException extends RuntimeException {
+
+    private static final String INTRO = "Nyat a valid instruction! Rub my belly instead!\n";
+    private static final String TODO = "Input 'todo ABC' to add task ABC\n";
+    private static final String EVENT = "Input 'event ABC /at DATE' to add event ABC on"
+            + " DATE\n";
+    private static final String DEADLINE = "Input 'deadline ABC /by DATE' to add deadline"
+            + " ABC due by DATE\n";
+    private static final String MARK = "Input 'mark xxx' to mark task xxx as complete\n";
+    private static final String UNMARK = "Input 'unmark xxx' to mark task xxx as incomplete\n";
+    private static final String DELETE = "Input 'delete xxx' to delete task xxx from the list\n";
+    private static final String LIST = "Input 'list' for overview\n";
+    private static final String BYE = "Input 'bye' to exit.\n";
+    private static final String OUTRO = "NYAAAAAA!\n";
+    private static final String TAB = "    ";
```

Comment:
What about using \t instead of the empty spaces for formatting? 🤔

Reply:
Oo I didn't know about that, thanks